### ISSUE TITLE
Drag & drop accepts all types of files skipping validations

### ISSUE THREAD

**Issue by jniechcial** [funkensturm/ember-cli-file-picker#4]
Drag&Drop way of handling upload accepts all types of files skipping the validation defined in `accept` property. 

Current workaround is to manually validate files once again on action `fileLoaded`. Don't know if it will be a good solution to implement inside the addon, what do you think?

**Comment by fsmanuel** [comment]
@jniechcial you are right. accepts only works for the browser dialog. I'll add instructions for manuall validation in the readme.

**Comment by jniechcial** [comment]
And what do you think about validating it inside the component before sending `fileLoaded` action? The component could fill in `errors` and through this binding inform the user that the file has not beed uploaded. The `fileLoaded` action would never be triggered.

**Comment by fsmanuel** [comment]
I don't know the best way to validate. I would suggest to move the component into the addon namespace. That way you can import the component and add the validation method you like.
`handleFiles` would look like that:
```js
  handleFiles: function(files) {
    if (this.validateFiles === 'function') {
      if (!this.validateFiles(files)) {
        return;
      }
    }

    // ...
  }
```
That way you can add what ever validations you like and add the errors you want. What do you think?

**Comment by jniechcial** [comment]
This is great. This way the user could validate upon name, type or the size on his own terms. Totally agree.

**Comment by fsmanuel** [comment]
@jniechcial can you test against master:
https://github.com/funkensturm/ember-cli-file-picker#validations

**Comment by fsmanuel** [comment]
@jniechcial i published `0.0.5` if you have problems please reopen.

**Comment by jniechcial** [comment]
@fsmanuel Thanks for your work. I will test it at the end of this week probably. As you said - if anything turnes out, I'll reopen.
